apps/webapp/app/components/runs/v3/RunFilters.tsx

```
@@ -355,8 +394,9 @@ function FilterMenu(props: RunFiltersProps) {
       variant={"secondary/small"}
       shortcut={shortcut}
       tooltipTitle={"Filter runs"}
+      className="pr-0.5"
     >
-      Filter
+      <></>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)